packages/trader/src/App/Components/Elements/Errors/error-component.tsx

```diff
@@ -29,22 +37,12 @@ const ErrorComponent = ({
     return (
         <PageErrorContainer
             error_header={header ?? ''}
-            error_messages={message ? message[(message, refresh_message)] : []}
+            error_messages={message ? [message, refresh_message] : []}
```

Comment:
This change is going to break some of the newly added error messages. 
Any reason behind this change?

ref: https://github.com/binary-com/deriv-app/issues/8333